pkg/launcher/options.go

```diff
@@ -55,6 +55,8 @@ type Options struct {
 	AutoupdateInterval time.Duration
 	// UpdateChannel is the channel to pull options from (stable, beta, nightly).
 	UpdateChannel autoupdate.UpdateChannel
+	// The GUN prefix for where the launcher and osqueryd collections are
```

Comment:
GUNPrefix is the path prefix used to store launcher and osqueryd binaries on the Notary server